sectionproperties/pre/library/concrete_sections.py

```diff
@@ -170,10 +170,11 @@ def concrete_column_section(
     :raises ValueErorr: If the number of bars in either 'n_bars_b' or 'n_bars_d' is not greater 
         than or equal to 2.
 
-    The following example creates a 600D x 300W concrete beam with 3N20 bottom steel
-    reinforcing bars and 30 mm cover::
+    The following example creates a 600D x 300W concrete column with 25 mm diameter
+    reinforcing bars each with 500 mm**2 area and 35 mm cover in a 3x6 array without
```

Comment:
for this example should the bar area be corrected to 491mm^2 for a 25 mm diameter metric bar?

Reply:
I am intending to use both a nominal diameter and nominal area to demonstrate that the area and diameter inputs are decoupled.